remote/package.json

```diff
@@ -52,7 +52,7 @@
     "vscode-regexpp": "^3.1.0",
     "vscode-textmate": "9.0.0",
     "xterm": "5.2.0-beta.49",
-    "xterm-addon-canvas": "0.4.0-beta.14",
+    "xterm-addon-canvas": "github:lewis-sanchez/xterm.js#0.4.2",
```

Comment:
Could you please open a follow-up issue to track switching back to the official package once it's published?